src/instance-validator.js

```diff
@@ -7,7 +7,7 @@ import { SequelizeMethod } from './utils/sequelize-method';
 const _ = require('lodash');
 const sequelizeError = require('./errors');
 const { BelongsTo } = require('./associations/belongs-to');
-const validator = require('./utils/validator-extras').validator;
+const validator = require('./utils/validator-extras').Validator;
```

Comment:
Why not use `{ Validator }` here like in `sequelize.js`?

Reply:
> Why not use `{ Validator }` here like in `sequelize.js`?

Makes sense!